GPU/Common/FramebufferManagerCommon.cpp

```diff
@@ -538,28 +528,36 @@ void FramebufferManagerCommon::CopyToDepthFromOverlappingFramebuffers(VirtualFra
 			continue;
 
 		if (src->fb_address == dest->z_address && src->fb_stride == dest->z_stride && src->format == GE_FORMAT_565) {
-			if (src->colorBindSeq > dest->depthBindSeq) {
-				// Source has older data than the current buffer, ignore.
-				continue;
+			if (src->colorBindSeq < dest->depthBindSeq) {
+				// Source has newer data than the current buffer, use it.
+				sources.push_back(CopySource{ src, RASTER_COLOR });
 			}
-			sources.push_back(CopySource{ src, RASTER_COLOR });
 		} else if (src->z_address == dest->z_address && src->z_stride == dest->z_stride && src->depthBindSeq > dest->depthBindSeq) {
 			sources.push_back(CopySource{ src, RASTER_DEPTH });
 		} else {
 			// TODO: Do more detailed overlap checks here.
 		}
 	}
 
-	// TODO: A full depth copy will overwrite anything else. So we can eliminate
+	std::sort(sources.begin(), sources.end());
+
+	// TODO: A full copy will overwrite anything else. So we can eliminate
 	// anything that comes before such a copy.
 
-	for (auto &source : sources) {
+	// For now, let's just do the last thing, if there are multiple.
+
+	// for (auto &source : sources) {
+	if (sources.size()) {
+		auto &source = sources.back();
 		if (source.channel == RASTER_DEPTH) {
 			// Good old depth->depth copy.
 			BlitFramebufferDepth(source.vfb, dest);
-
 			gpuStats.numDepthCopies++;
 			dest->last_frame_depth_updated = gpuStats.numFlips;
+		} else if (source.channel == RASTER_COLOR) {
+			VirtualFramebuffer *src = source.vfb;
+			// Copying color to depth.
+			BlitUsingRaster(src->fbo, 0.0f, 0.0f, src->renderWidth, src->renderHeight, dest->fbo, 0.0f, 0.0f, dest->renderWidth, dest->renderHeight, false, DRAW2D_565_TO_DEPTH, "565_to_depth");
```

Comment:
nit: Maybe log drawnFormat if not 565?  Maybe related to the Colin thing?

-[Unknown]

Reply:
I'm planning to remove drawnFormat soon, and instead have duplicate VirtualFramebuffer, and automatic reinterpret copies between them on-demand. But yeah, can certainly add such logging in the meantime.